sound/soc/sof/sof-priv.h

```diff
@@ -386,6 +386,7 @@ struct snd_sof_dev {
 	struct snd_sof_mailbox dsp_box;		/* DSP initiated IPC */
 	struct snd_sof_mailbox host_box;	/* Host initiated IPC */
 	struct snd_sof_mailbox stream_box;	/* Stream position update */
+	struct snd_sof_mailbox debug_box;	/* Debug initiated IPC */
 	struct snd_sof_ipc_msg *msg;
 	int ipc_irq;
```

Comment:
Comment here can be enhanced. 'Debug initiated IPC' doesn't make sense because Debug is not an entity who can do stuff like DSP or Host as we see in the above comments for dsp_box and host_box.
We can either remove this comment or say ``` /* debug info updates */```